src/graphics/text.rs

```diff
@@ -295,43 +303,38 @@ const DEFAULT_TEXT_BUFFER_SIZE: usize = 64;
 #[derive(Debug)]
 pub struct Text {
     batch: SpriteBatch,
-    atlas: Cached<FontAtlas>,
 }
 
 impl Text {
-    pub fn from_cached(ctx: &mut Graphics, font_atlas: Cached<FontAtlas>) -> Self {
-        Self::from_cached_with_capacity(ctx, font_atlas, DEFAULT_TEXT_BUFFER_SIZE)
+    pub fn from_cached<T: Clone + Into<Cached<Texture>>>(ctx: &mut Graphics, texture: T) -> Self {
```

Comment:
`Clone` bound is unnecessary

Reply:
also, `from_cached` should probably be `new` here; worth noting that now that we have the `Into<Cached<Texture>>` bound, you can use a raw `Texture` here as well, so it's no longer purely from `Cached`